### DISCUSSION TITLE
Find out if any part of https://github.com/google/lyra may be helpful for this project.

### DISCUSSION THREAD

**Discussion by KOLANICH** [RHVoice/RHVoice#279] [Ideas]

**Comment by alex19EP** [comment]
can you express your idea in more detail?
so far I do not understand how this can be useful for us.

**Reply by KOLANICH** [reply]
That piece of software contains some optimized code related to neural networks (and a pretrained network too) and routines for signal processing, that likely can be used in some parametric TTSes and vocoders. IDK how well it all aligns with RHVoice internals and architecture and future plans though.

**Reply by abitrolly** [reply]
There are a lot of software pieces that contain optimized code. Just because it is written by Google doesn't mean it will be useful everywhere. Maybe it is time for you to start reading about HMM based synthesis and ask relevant questions.
